virtcontainers/clh.go

```diff
@@ -861,439 +784,165 @@ func (clh *cloudHypervisor) LaunchClh() (string, int, error) {
 //###########################################################################
 
 const (
-	cctOFF  string = "off"
-	cctFILE string = "file"
+	cctOFF  string = "Off"
```

Comment:
Why? Is this a CLH change since the previous version (I still say why btw though :smile:)?

Reply:
it is just to match the the API definition